ChemistryLib/PhreeqcInterfaceData/EquilibriumPhase.h

```diff
@@ -0,0 +1,46 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2019, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#pragma once
+
+#include <boost/optional/optional.hpp>
+#include <fstream>
+#include <string>
+#include <vector>
+
+namespace BaseLib
+{
+class ConfigTree;
+}
+
+namespace ChemistryLib
+{
+struct EquilibriumPhase
+{
+    EquilibriumPhase(std::string name_,
+                     double amount_,
+                     double saturation_index_)
+        : name(std::move(name_)),
+          amount(amount_),
+          saturation_index(saturation_index_)
+    {
+    }
+
+    friend std::ofstream& operator<<(
+        std::ofstream& out,
+        std::vector<EquilibriumPhase> const& equilibrium_phases);
+
+    std::string const name;
+    double amount;
```

Comment:
const?

Reply:
in `std::ifstream& operator>>(std::ifstream& in, phreeqcIO& phreeqc_io)` (see ChemistryLib/PhreeqcInterface.cpp)
```
                case ItemType::EquilibriumPhase:
                {
                    // Update amounts of equilibrium phases
                    auto& equilibrium_phase = BaseLib::findElementOrError(
                        equilibrium_phases.begin(), equilibrium_phases.end(),
                        [&item_name](
                            EquilibriumPhase const& equilibrium_phase) {
                            return equilibrium_phase.name == item_name;
                        },
                        "Could not find equilibrium phase '" + item_name +
                            "'.");
                    equilibrium_phase.amount = accepted_items[item_id];
                    break;
```
the amount of an equilibrium phase has to be updated after the chemical calculation by Phreeqc.